site/_includes/partials/script.js

```diff
@@ -38,7 +38,22 @@
     '{{ analytics.dimensions.TRACKING_VERSION }}',
     '{{ analytics.TRACKING_VERSION }}'
   );
+
   ga('send', 'pageview');
+  const chromeAnchors = document.getElementsByTagName('a');
```

Comment:
I am not sure what this script is intended to do, but any code that is required for `/100/` should just be added to that page specifically, and not "leak" out into partials that are used on the rest of the site.

Reply:
Moved to /100/ directory.

Reply:
It is not moved, I can still see it here. 

It looks like you try to attach analytics events to clicks throughthough the page. I think it can stay, but with some modifications:

1. Create one event listener for whole document and not 100s of listeners for each link. Use event.target. event delegation and event.target to fire listener for the right element.

2. Don't send data for every <a> click, this will create lots of noise. Add a class name, .e.g 'ga-event' only to the elements you want to track.

3. Use more specific data- attributes, as e.g. data-category is already used on dcc in other context. Use e.g. data-ga-action, data-ga-category. Make sure they exist before sending an empty hit.

4. Send value too (can be empty if not relevant).

Example implementation: https://github.com/GoogleChrome/web.dev/blob/main/src/lib/analytics.js#L93